x-pack/plugins/security_solution/common/detection_engine/rule_management/api/rules/bulk_actions/request_schema.ts

```diff
@@ -164,12 +172,17 @@ export const PerformBulkActionRequestBody = t.intersection([
         action: t.union([
           t.literal(BulkActionType.delete),
           t.literal(BulkActionType.disable),
-          t.literal(BulkActionType.duplicate),
           t.literal(BulkActionType.enable),
           t.literal(BulkActionType.export),
         ]),
       })
     ),
+    t.exact(
+      t.type({
+        action: t.literal(BulkActionType.duplicate),
+        [BulkActionType.duplicate]: bulkActionDuplicatePayload,
+      })
+    ),
```

Comment:
This would be a breaking change in the API because the payload is now required. To keep backward compatibility, we should make it optional and preserve the current behavior when it is not passed.